dev/Dockerfile.base

```diff
@@ -75,19 +77,19 @@ RUN set -ex; \
 
 # Finalize installation
 RUN set -ex; \
-    mkdir -p /var/lib/pulp/artifact \
+    mkdir -p /entrypoints.d \
+            /etc/ansible \
+    && mkdir --mode=2775 -p /var/lib/pulp \
+    && chown ${USER_NAME}:${USER_GROUP} /var/lib/pulp \
+    && mkdir --mode=2775 -p \
+             /var/lib/pulp/artifact \
              /var/lib/pulp/tmp \
              /tmp/ansible \
-             /etc/ansible \
-             /entrypoints.d \
-    && chown -R ${USER_NAME}:${USER_GROUP} \
-        /var/lib/pulp \
+    && chown ${USER_NAME}:${USER_GROUP} \
         /tmp/ansible \
         /etc/ansible \
-    && chmod -R 0775 /var/lib/pulp \
-                     /app/docker/entrypoint.sh \
+    && chmod -R 2775 /app/docker/entrypoint.sh \
```

Comment:
Not sure about this one, do we want g+s on a shell script here?

Reply:
good catch. Looks like I got overzealous when fixing up the merge. Will update.

Reply:
Updated with a fix for this chmod.